examples/fluids/src/setuplibceed.c

```diff
@@ -264,6 +264,27 @@ PetscErrorCode SetupLibceed(Ceed ceed, CeedData ceed_data, DM dm, User user, App
   CeedQFunctionAddInput(ceed_data->qf_ics, "qdata", q_data_size_vol, CEED_EVAL_NONE);
   CeedQFunctionAddOutput(ceed_data->qf_ics, "q0", num_comp_q, CEED_EVAL_NONE);
 
+  // -- Create QFunction for computing the L2 error
+  if (problem->non_zero_time && user->app_ctx->test_type == TESTTYPE_NONE) {
```

Comment:
Side note; I'm not sure what `non_zero_time` means.

Reply:
I don't think that this is a very descriptive name but it is a flag for problems with exact solutions. Maybe I can change it to `has_true_soln`?! I think the channel problem is missing it. Beside blasius and channel (NEWTONIAN), 2D advection and traveling vortex have exact solutions as well.

Reply:
Well, I have broken the functionality we previously had for the euler and advection solvers!!!

Reply:
If the flag is for problems with exact solutions, I think `has_true_soln` works much better.

Reply:
Hmm, the channel problem doesn't work by simply turning on the flag. We could come back to this later!